.github/workflows/build.yml

```diff
@@ -12,36 +12,36 @@ env:
 jobs:
   docker:
     name: docker build amd64
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
ensures same os version as in aws

Reply:
will it become unsupported in 3 months or so?

Reply:
it's an LTS (long term support release) and it'd be supported for 6 years iirc